# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libcap-ng/libcap-ng-include_sys_types.h.patch
# Copyright (C) 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

pid_t and uid_t are defined in sys/types.h

Note: Bug detected when building against musl libc.

--- a/utils/proc-llist.h	2014-04-01 22:53:46.970320837 +0200
+++ b/utils/proc-llist.h	2014-04-01 22:54:41.227573407 +0200
@@ -26,6 +26,7 @@
 
 #include "config.h"
 
+#include <sys/types.h> /* for pid_t, uid_t */
 
 /* This is the node of the linked list. Any data elements that are per
  *  record goes here. */
